File: Task1-Server/Kolosok.Domain/Entities/Action.cs
namespace Kolosok.Domain.Entities;

public class Action : BaseEntity
{
    public string Name { get; set; }
    public string Description { get; set; }
    public string ActionTime { get; set; }
    public string ActionType { get; set; }
    public string ActionPlace { get; set; }
    
    public Guid BrigadeRescuerId { get; set; }
    public BrigadeRescuer BrigadeRescuer { get; set; }
    
    public Guid VictimId { get; set; }
    public Victim Victim { get; set; }
}

File: Task3-WebClient/src/app/models/action.ts
import {Victim} from "./victim";
import {BrigadeRescuer} from "./brigade-rescuer";
import {BaseEntity} from "./base-entity";

export interface Action extends BaseEntity {
  name: string;
  description: string;
  actionTime: string;
  actionType: string;
  actionPlace: string;
  brigadeRescuerId: string;
  brigadeRescuer: BrigadeRescuer;
  victimId: string;
  victim: Victim;
}
